src/include/cursor.h

```diff
@@ -413,7 +416,9 @@ struct __wt_cursor_log {
 	uint32_t	step_count;	/* Intra-record count */
 	uint32_t	rectype;	/* Record type */
 	uint64_t	txnid;		/* Record txnid */
-	uint32_t	flags;
+
+#define	WT_CURLOG_ARCHIVE_LOCK	0x01	/* Archive lock held */
+	u_int		flags;
```

Comment:
Sure, done & pushed. (I used uint8_t because that's what the other cursors used.)